doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -186,56 +195,52 @@ analysis on inductive or co-inductive objects (see :ref:`inductive-definitions`)
       clause is an occurrence clause whose syntax and behavior is described
       in :ref:`occurrences sets <occurrencessets>`.
 
-   .. tacv:: destruct @term {? with @bindings } {? as @or_and_intropattern_loc } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
-             edestruct @term {? with @bindings } {? as @or_and_intropattern_loc } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
+   .. tacv:: destruct @term {? with @bindings } {? @as_or_and_ipat } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
+             edestruct @term {? with @bindings } {? @as_or_and_ipat } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
 
       These are the general forms of :tacn:`destruct` and :tacn:`edestruct`.
       They combine the effects of the ``with``, ``as``, ``eqn:``, ``using``,
       and ``in`` clauses.
 
-.. tacn:: case @term
-   :name: case
+.. tacn:: case @induction_clause_list
 
    The tactic :n:`case` is a more basic tactic to perform case analysis without
-   recursion. It behaves as :n:`elim @term` but using a case-analysis
-   elimination principle and not a recursive one.
+   recursion. It behaves like :n:`elim @term` but uses a case-analysis
```

Comment:
> @Zimmi48 Deprecate tactics that are not used anymore: `cut`, `elim`, `case` (vanilla versions), as well as `simple destruct` [from [#8946](https://github.com/coq/coq/issues/8946#issue-378672977)]

Is this a good time to deprecate `simple destruct`, which has only 9 uses in the standard library?  OTOH, I see ~3000 of uses of `elim` and ~800 of `cut`--do you really want to deprecate these (or recommend using other tactics in their place)?

If replacing of any of 4 mentioned tactics is mostly mechanical, I might create PRs to remove them for 8.15 though I might well need help for exceptional uses.

Reply:
Why not raising the question of deprecating `simple destruct`. `cut` can really be replaced by `assert` or `enough` so it could make sense to deprecate it in theory, but it would be more work (I'm not sure how easy it would be to do this mechanically). It doesn't seem like `elim` can be deprecated at this time, but we can document that its use is discouraged for basic users.